index.html

```diff
@@ -36,35 +36,35 @@
 
     <body>
         <div class="body-container">
-
             <!-- Navbar -->
-            <nav>
-                <a href="#technologies">Technologies</a>
-                <a href="#projects">Projects</a>
-                <a href="#skills">Other Skills</a>
-                <a href="#contacts">Contacts</a>
+            <nav class="nav">
+                <a class="nav__link" href="#technologies">Technologies</a>
+                <a class="nav__link" href="#projects">Projects</a>
+                <a class="nav__link" href="#skills">Other Skills</a>
+                <a class="nav__link" href="#contacts">Contacts</a>
             </nav>
-            
+
             <!-- Header -->
-            
-            <header>
-                <div class="header-text">
+
+            <header class="header">
+                <div class="header__wrapper">
```

Comment:
„Wrapper“ je něco kolem a toto je uvnitř.
Vymysli lepší jméno.
```suggestion
                <div class="header__content">
```